src/App.js

```diff
@@ -1,23 +1,148 @@
-import React from 'react';
-import SentenceContainer from './components/Sentence/SentenceContainer.js'
-import SettingsContainer from './components/SettingsContainer.js'
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import Sentence from './components/Sentence/Sentence'
+import Settings from './components/Settings'
 import './App.css';
+import {allScales, getAccidentalFromNoteString, isInArray, keyFormulas, nashvilleNumbers} from "./Constants";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-          <SentenceContainer/>
-      </header>
-        <div>
-            TODO:
-            <li>- SHow guess value in setence with green or red correct/incorrect</li>
-                <li>- settings/config</li>
-        </div>
-        <SettingsContainer/>
-    </div>
-  );
+class App extends Component
+{
+
+    getAllKeysWithTrueValue = (obj) => {
```

Comment:
This type of function seems like it can be refactored out. It is not necessary now but I typically put this sort of stuff in `lib/helpers/someHelper.js`

Reply:
https://github.com/tmbridge/chords-in-keys/pull/2/files#r282015859 ?

I'm using Constants.js but not a bad idea to make another for helpers/util methods.